### PR TITLE
3 d volume search

### PR DESCRIPTION
**Pull request by annie-xd-wang**

### CODE REVIEW

**Comment by annie-xd-wang** [comment]
We can start testing with the feature pipeline: [{"name": ZStackAcquisition,"args": (False,False,"z-stack",True,),},{"name": WaitToContinue,},{"name": VolumeSearch3D,"args": ("Nanoscale","N/A",0,1,0.5,None,),},{"name": WaitToContinue,},{"name": ChangeResolution,"args": ("Nanoscale","N/A",),},{"name": ZStackAcquisition,"args": (False,False,"z-stack",True,),},]. We need to load a custom function to accurately label the images for improved results.

**Comment by AdvancedImagingUTSW** [comment]
Is this ready for review @annie-xd-wang ?

**Comment by annie-xd-wang** [comment]
Yes, it is ready for review.

File: src/navigate/model/data_sources/tiff_data_source.py [hunk 1]
```diff
@@ -153,6 +159,46 @@ def read(self) -> None:
...
+            values from 1, 2, 4, 8
+            Not supported for now.
+
+        Returns
+        -------
+        data : npt.ArrayLike
+            Image data
+        """
+        # TODO: may need to support .tif
+        file_suffix = ".ome.tiff" if self.is_ome else ".tiff"
```
Comment (AdvancedImagingUTSW) [inline src/navigate/model/data_sources/tiff_data_source.py:186]: Why don't we use a try/except/finally statement to also support ".tif"?

**Review by AdvancedImagingUTSW — APPROVED** [review]

**Pull request merged**